consensus/src/network.rs

```diff
@@ -701,164 +708,168 @@ impl NetworkTask {
+                                    event = LogEvent::ReceiveBlockRetrieval,
+                                    "{}",
+                                    request
                                 );
-                                info!(
-                                    LogSchema::new(LogEvent::NetworkReceiveProposal)
-                                        .remote_peer(peer_id),
-                                    block_round = proposal.proposal().round(),
-                                    block_hash = proposal.proposal().id(),
+                                IncomingRpcRequest::BlockRetrieval(IncomingBlockRetrievalRequest {
+                                    req: *request,
+                                    protocol,
+                                    response_sender: callback,
+                                })
+                            },
+                            ConsensusMsg::BatchRequestMsg(request) => {
+                                debug!(
+                                    remote_peer = peer_id,
+                                    event = LogEvent::ReceiveBatchRetrieval,
+                                    "{}",
+                                    request
                                 );
-                            }
-                            Self::push_msg(peer_id, consensus_msg, &self.consensus_messages_tx);
-                        },
-                        // TODO: get rid of the rpc dummy value
-                        ConsensusMsg::RandGenMessage(req) => {
-                            let (tx, _rx) = oneshot::channel();
-                            let req_with_callback =
+                                IncomingRpcRequest::BatchRetrieval(IncomingBatchRetrievalRequest {
+                                    req: *request,
+                                    protocol,
+                                    response_sender: callback,
+                                })
+                            },
+                            ConsensusMsg::DAGMessage(req) => {
+                                IncomingRpcRequest::DAGRequest(IncomingDAGRequest {
+                                    req,
+                                    sender: peer_id,
+                                    responder: RpcResponder {
+                                        protocol,
+                                        response_sender: callback,
+                                    },
+                                })
+                            },
+                            ConsensusMsg::CommitMessage(req) => {
+                                IncomingRpcRequest::CommitRequest(IncomingCommitRequest {
+                                    req: *req,
+                                    protocol,
+                                    response_sender: callback,
+                                })
+                            },
+                            ConsensusMsg::RandGenMessage(req) => {
```

Comment:
hmm, is this block just formatting change?

Reply:
I added `RoundTimeoutMsg` handling, and then the formatter took over.